### PR TITLE
TensorBoard 2.11.0 release notes.

### PR DESCRIPTION
**Pull request by JamesHollyer**
Release notes for the TensorBoard 2.11.0 release.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Release 2.11.0
+
+The 2.11 minor series tracks TensorFlow 2.11.
+
+## Bug Fixes
+
+- Fixed HParams plugin sorting bugs (#5925)
```
Comment (nfelt) [inline RELEASE.md:7]: I think this is the wrong PR - it should actually be go/tbpr/5971 I believe?

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+- Fix color by menu drop down bug in projector plugin (#5974)
+- Fix bug with histogram axis being off due to rounding (#5925)
+
+## Breaking Changes
+
+- Tensorboard now only supports Python 3.7 and above (#5878)
```
Comment (nfelt) [inline RELEASE.md:13]: Nit: let's capitalize the B, TensorBoard

**Review by nfelt — APPROVED** [review]
Couple minor things, otherwise LGTM

File: RELEASE.md (hunk 2 above)
Comment (JamesHollyer) [reply RELEASE.md]: Unacceptable. I should be ashamed! :P

File: RELEASE.md (hunk 1 above)
Comment (JamesHollyer) [reply RELEASE.md]: Whoops. Must have been a copy paste error. Good catch. Fixed.

**Comment by JamesHollyer** [comment]
Yes

**Pull request closed without merging**